src/main/java/duke/command/SaveCommand.java

```diff
@@ -0,0 +1,17 @@
+package duke.command;
+
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.UI;
+
+public class SaveCommand implements Command {
+    @Override
+    public String execute(TaskList taskList, UI ui, Storage storage) {
+        return storage.saveData(ui, taskList);
+    }
+
+    @Override
+    public boolean isExit() {
```

Comment:
Missing javadocs for public method

Reply:
Hi Clara thanks for your review, maybe next time when reviewing a pull request consider not copy and pasting the same comments over and over? I understood what you meant the first few times :)